int/ks-devops-ext-tekton-common/config/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "app": {
+    "name": "common-tekton-manifests",
+    "namespace": "ks-devops-ext",
+    "path": "prod/ks-devops-ext-tekton-common",
+    "clusterName": "in-cluster"
```

Comment:
All the Argo CD applications are stored in the same cluster. So, we need to avoid the naming conflict.

And the correct clusterName should be `int`.

See also https://github.com/kubesphere-sigs/ks-infra/blob/master/int/ks-releaser/config.json